### PR TITLE
Add all of the missing RPC apis

### PR DESCRIPTION
**Pull request by liorbond**

### CODE REVIEW

File: proto/secret/compute/v1beta1/query.proto [hunk 1]
```diff
@@ -34,10 +34,24 @@ service Query {
...
+    // Query contract hash by address
+    rpc ContractHash (QueryContractHashRequest) returns (QueryContractHashResponse) {
+        option (google.api.http).get = "/compute/v1beta1/hash/address/{address}";
+    }
     // Query contract hash by code id
     rpc ContractHashByID (QueryContractHashByIDRequest) returns (QueryContractHashResponse) {
         option (google.api.http).get = "/compute/v1beta1/hash/id/{code_id}";
     }
+    // Query contract key by address
+    rpc ContractKey(QueryContractKeyRequest) returns (QueryContractKeyResponse) {
```
Comment (assafmo) [inline proto/secret/compute/v1beta1/query.proto:46]: Irrelevant here

File: proto/secret/compute/v1beta1/query.proto [hunk 2]
```diff
@@ -34,10 +34,24 @@ service Query {
...
+        option (google.api.http).get = "/compute/v1beta1/key/{address}";
+    }
+    // Query contract address by label
+    rpc AddressByLabel(QueryContractAddressByLabelRequest) returns (QueryContractAddressByLabelResponse) {
```
Comment (assafmo) [inline proto/secret/compute/v1beta1/query.proto:50]: Should also add the other way around

File: proto/secret/compute/v1beta1/query.proto (hunk 1 above)
Comment (liorbond) [reply proto/secret/compute/v1beta1/query.proto:46]: Why? it was implemented in the querier but was missed here, If it's irrelevant ill remove it from the querier as well.

File: proto/secret/compute/v1beta1/query.proto (hunk 2 above)
Comment (liorbond) [reply proto/secret/compute/v1beta1/query.proto:50]: Working on it in another PR, this PR focus on those APIs that were implemented in the queried but weren't exported by the pb

**Review by Cashmaney — APPROVED** [review]

File: proto/secret/compute/v1beta1/query.proto (hunk 1 above)
Comment (assafmo) [reply proto/secret/compute/v1beta1/query.proto:46]: Because it's used only inside the enclave, there's no reason to use it outside

**Comment by assafmo** [comment]
Replaced by https://github.com/scrtlabs/SecretNetwork/pull/1130

**Pull request merged**